docs/_clef/CliqueSigning.md

```diff
@@ -66,16 +77,16 @@ INFO [06-16|11:14:54.125] Persisted trie from memory database      nodes=1 size=
 INFO [06-16|11:14:54.125] Successfully wrote genesis state         database=lightchaindata hash=187412..4deb98
 ```
 
-At this point, we have a Clique network which we can start sealing on.
+At this point a Clique network exists with blocks that can be signed.
 
-## Part 1: Prepping Clef
+## Prepping Clef
 
-In order to make use of `clef` for signing, we need to do a couple of things.
+In order to make use of Clef for signing:
```

Comment:
```suggestion
In order to make use of `clef` for signing:
```
When we talk about the binary itself, I think we should use `clef`. I may have used "Clef" here and there, but that's not right IMO

Reply:
ok!